jdaviz/configs/rampviz/plugins/viewers.py

```diff
@@ -36,21 +36,29 @@ def _initialize_x_axis(self):
             self.reset_limits()
 
     def reset_limits(self):
-        super().reset_limits()
-
         # override to reset to the global y limits including marks:
-        global_y_min = float(self.state.y_min)
-        global_y_max = float(self.state.y_max)
+        global_y_min = np.inf
+        global_y_max = -np.inf
         for mark in self.figure.marks:
             if len(mark.y) and mark.visible:
                 global_y_min = min(global_y_min, np.nanmin(mark.y))
                 global_y_max = max(global_y_max, np.nanmax(mark.y))
 
-        if global_y_min != self.state.y_min or global_y_max != self.state.y_max:
-            self.set_limits(
-                y_min=global_y_min * 0.9,
-                y_max=global_y_max * 1.1
-            )
+        for layer in self.state.layers:
+            if not layer.layer.label.startswith('Subset') and layer.visible:
```

Comment:
can this check if the layer is a subset rather than relying on a label check?  (It would be possible to name a data layer starting with subset and eventually we want to support renaming subsets)

Reply:
Deal.